ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.less

```diff
@@ -62,14 +59,18 @@
 
 @{customElementNamespace}filter.is-location {
   > .filter-input {
-    margin-top: @minimumSpacing;
     display: block;
+    padding: @minimumSpacing 1.5rem 0px @minimumSpacing + 0.75 *
```

Comment:
Does that last statement work? I thought normally when doing operations in css you had to wrap in a `calc()`